spec/examples/formulas/work_sans_macos_only.yml

```diff
@@ -0,0 +1,281 @@
+---
+name: Work Sans
+description: Work Sans
+homepage: http://wweeiihhuuaanngg@gmail.com
```

Comment:
This address is a mistake?

Reply:
Checked, it's an intended behavior. `fontist` takes information on homepage from the `Vendor URL` of `otfinfo`, and it contains the email:

```
$ otfinfo -i ~/.fontist/fonts/WorkSans-Regular.ttf
Family:              Work Sans
Subfamily:           Regular
Full name:           Work Sans Regular
PostScript name:     WorkSans-Regular
Version:             Version 2.010; ttfautohint (v1.8.3)
Unique ID:           2.010;WEI;WorkSans-Regular
Designer:            Wei Huang
Designer URL:        http://wweeiihhuuaanngg@gmail.com
Manufacturer:        Wei Huang
Vendor URL:          http://wweeiihhuuaanngg@gmail.com
Copyright:           Copyright 2019 The Work Sans Project Authors (https://github.com/weiweihuanghuang/Work-Sans)
License URL:         http://scripts.sil.org/OFL
License Description: This Font Software is licensed under the SIL Open Font License, Version 1.1. This license is available with a FAQ at: http://scripts.sil.org/OFL
Vendor ID:           WEI
```

Reply:
Thanks @alexeymorozov , this is hilarious 😆